R/isoread_rds.R

```diff
@@ -20,33 +18,33 @@ iso_read_rda <- function(ds) {
     })
     iso_files <- iso_as_file_list(isofiles)
   }
-  if (!exists("iso_files", inherits = FALSE) || !(iso_is_object(iso_files))) 
+  if (!exists("iso_files", inherits = FALSE) || !(iso_is_object(iso_files)))
```

Comment:
first part of the if statement not necessary because of the explicit assignment to `iso_files <- readRDS(...)` earlier